incydr/cli/cmds/options/utils.py

```diff
@@ -0,0 +1,12 @@
+def user_lookup(ctx, param, value):
```

Comment:
Call function to lookup user ID by username.  I assumed if the value contained `@` then we could assume it was an email/username.  If you have a better idea for how to differentiate them lmk!

Reply:
Nope, this works.